### PR TITLE
Prince/create a workflow to update translators branch

### PR DESCRIPTION
**Pull request by prince-deriv**
## Changes:

Added a workflow to keep the `translators` branch up to date so translators can pick up and proceed with the translation process

### CODE REVIEW

**Review by aswathy-deriv — DISMISSED** [review]

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6770947261) |

**Comment by DerivFE** [comment]
**Preview Link**: https://prince-automate-tran.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://prince-automate-tran.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6770947261) |

File: .github/workflows/translators-sync.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  push:
+    branches:
+        - 'master*'
+        
+jobs:
+    sync-translation:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout 🛎️
+              uses: actions/checkout@v2.3.1
```
Comment (markwylde-deriv) [inline .github/workflows/translators-sync.yml:21]:
```suggestion
              uses: actions/checkout@v4.1.1
```

Latest version is v4.1.1
https://github.com/actions/checkout

File: .github/workflows/translators-sync.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            - name: Setup Node
+              uses: actions/setup-node@v2.1.2
```
Comment (markwylde-deriv) [inline .github/workflows/translators-sync.yml:23]:
```suggestion
              uses: actions/setup-node@v4.0.0
```

https://github.com/actions/setup-node

**Review by markwylde-deriv — DISMISSED** [review]

**Comment by DerivFE** [comment]
**Preview Link**: https://prince-automate-tran.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://prince-automate-tran.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6771234445) |

File: .github/workflows/translators-sync.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                  git config --global user.name "DerivFE"
+                  git config --global user.email "80095553+DerivFE@users.noreply.github.com"
+
+                  echo "Checking out new branch [$branch_name]"
+                  git checkout -b "$branch_name"
+
+                  # Force push to this branch in case a previous run created it.
+                  git push --set-upstream origin "$branch_name" -f
+
+                  sudo apt install gh
```
Comment (habib-deriv) [inline .github/workflows/translators-sync.yml:40]: Can we add a check to not to install gh, if it's already installed?
```suggestion
                  sudo apt install gh
```

Comment (habib-deriv) [inline .github/workflows/translators-sync.yml:40]: Can we add a check to not to install gh, if it's already installed?

Comment (markwylde-deriv) [reply .github/workflows/translators-sync.yml:40]: @habib-deriv does it take a long time, even if it's already installed? For me (in docker) the first install of `gh` takes 3.2s, but then second call to install is 0.5s:

```
root@2a81a4fdf583:/# time apt install gh
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
gh is already the newest version (2.4.0+dfsg1-2).
0 upgraded, 0 newly installed, 0 to remove and 7 not upgraded.

real	0m0.557s
user	0m0.504s
sys	0m0.054s
```

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**